cilium/cmd/config_get.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/test/helpers"
+	"github.com/spf13/cobra"
+)
+
+var (
+	re = regexp.MustCompile("[^0-9a-zA-Z]")
```

Comment:
Wouldn't `[^\w]` work here?

Could you also rename the variable to better clarify its purpose?